CMSgov/design-system
# [WNMGDS-1188] filterchip a11y

## Summary
https://jira.cms.gov/browse/WNMGDS-1188

### Added
- `pointer-events: none` to `<button>` to prevent events from targeting wrong element
- `aria-live` and instructions for use to our documentation
- `aria-relevant` and instructions for use to our documentation

### Changed
Documentation
- Improved existing info
- Removed bad examples
- Made `ariaClearLabel` optional since we now provide a sensible default

### Fixed
Included Space in list of keycodes that control removing chip.

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1188/filterchip-a11y/components/filterchip/

Run our React FilterChip example on Safari + VoiceOver and use keyboard to navigate. `aria-label` should give you a clear idea what interacting with the filter will do, and you should hear an announcement once element leaves the page.
